medocker/docker_shared/run_warpx.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# This script handles running MPI/Warp in docker containers on AWS, and the
+# associated copy and monitoring operations that accompany it. It detects runs
+# that don't output to stdout for $TIMEOUT seconds and will terminate at that
+# point. It will move files either as the run is ongoing, or just at the end,
+# back to S3. Dump files are never moved or copied to S3; they will be copied
+# from S3 if they're already there.
+
+# S3 can sometimes make directories with no name ("") if two slashes are
+# present in sequence
+# https://stackoverflow.com/questions/1848415/remove-slash-from-the-end-of-a-variable
+DIRNAME=${1%/}
+BUCKET=$2
+SCRIPTNAME='run_script.sh'
+
+# MVFILES can be an environment variable set to True to move files as
+# generated.
+echo "Move diagnostic files as generated:" $MVFILES
+
+# TIMEOUT can be an environment variable. If no output to stdout has been made
+# in the previous TIMEOUT seconds, this script terminates.
+# TIMEOUT defaults to 3h = 10800 s
+# https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash
+if [ -z ${TIMEOUT:x} ] || ! (( "$TIMEOUT" > 0 )); then
+    echo TIMEOUT set to default of 10800 sec
+    TIMEOUT=10800
+else
+    echo TIMEOUT set to "$TIMEOUT"
+fi
+
+# Use EFS directory as working directory.
+echo Make directory ${DIRNAME} in EFS and cd there
+mkdir -p /efs/WarpX_simulation_runs/${DIRNAME}
+cd /efs/WarpX_simulation_runs/${DIRNAME}
+
+# Copy files from S3
+aws s3 cp --recursive s3://${BUCKET}/${DIRNAME} ./ --exclude "*" \
+    --include "run*" --include "*.dump" --include "std*" \
```

Comment:
`.dump` files are Warp specific and isn't (so far) used in WarpX.

Reply:
I can remove it, but it shouldn't be a problem, since the script just checks to see if any dump files exists and then copies them if they exist. We could keep this line in if we plan on using .dump files for warpx as well.

Reply:
The .dump format is specific to Warp so I don't see a way in which we would start using it but I agree it doesn't hurt anything.